tools/backups2datalad/datasetter.py

```diff
@@ -88,7 +89,8 @@ async def update_from_backup(
         if to_save:
             log.debug("Committing superdataset")
             superds.assert_no_duplicates_in_gitmodules()
-            await superds.save(message="CRON update", path=to_save)
+            msg = await self.get_superds_commit_message(superds)
```

Comment:
here I think it should be limited to `to_save` list of paths (used in `.save` below) since operation might be limited to specific dandisets iirc.

Reply:
Done: 0eb733a.